tasks/section5.yml

```diff
@@ -291,15 +417,19 @@
       - rule_5.2.7
 
 - name: "SCORED | 5.2.7 | PATCH | Ensure SSH HostbasedAuthentication is disabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^HostbasedAuthentication'
+      line: 'HostbasedAuthentication no'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.7
 
 - name: "SCORED | 5.2.8 | AUDIT | Ensure SSH root login is disabled"
-  command: /bin/true
+  command: grep "^PermitRootLogin no" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*